packages/modal/src/Modal.styles.ts

```diff
@@ -0,0 +1,146 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { fontFamilies, transitionDuration } from '@leafygreen-ui/tokens';
+import facepaint from 'facepaint';
+import { transparentize } from 'polished';
+import { CloseIconColor, ModalSize } from './types';
+
+// breakpoints for different screen sizes
+export const small = '767px'; // mobile screens, from 0px - 767px
+export const medium = '768px'; // tablet screens, from 768px - 1024px
+export const large = '1025px'; // laptops/desktop screens, from 1025px and above
+
+export const mq = facepaint([
+  `@media only screen and (max-width: ${small})`,
+  `@media only screen and (min-width: ${medium})`,
+  `@media only screen and (min-width: ${large})`,
+]);
+
+export const defaultHorizontalSpacing = 18;
+export const defaultVerticalSpacing = 64;
+
+export const backdropBaseStyle = css`
+  overflow-y: auto;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  opacity: 0;
+  transition: opacity ${transitionDuration.default}ms ease-in-out;
+`;
+
+export const backdropThemeStyles: Record<Theme, string> = {
+  [Theme.Light]: css`
+    background-color: ${transparentize(0.4, palette.black)};
+  `,
+  [Theme.Dark]: css`
+    background-color: ${transparentize(0.4, palette.gray.dark2)};
+  `,
+};
+
+export const visibleBackdrop = css`
+  opacity: 1;
+`;
+
+export const scrollContainer = css`
+  position: absolute;
+  min-height: 100%;
+  width: 100%;
+  padding: ${defaultVerticalSpacing}px ${defaultHorizontalSpacing}px;
+  overflow-y: auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+export const modalContentStyle = css`
+  font-family: ${fontFamilies.default};
+  transition: all ${transitionDuration.default}ms ease-in-out;
+  margin: auto;
+  max-height: calc(100% - ${defaultVerticalSpacing}px);
+  position: relative;
+  pointer-events: all;
+  transform: translate3d(0, -16px, 0);
+  opacity: 0;
+  border-radius: 24px;
+  padding: 40px 36px;
```

Comment:
nit, non-blocking but do we have tokens for these values?

Reply:
Do you mean the spacing tokens?

Reply:
yes

Reply:
I guess this is something else we can bring up during the API brainstorm, these numbers don't match the spacing tokens. It would only work if we did something like `spacing[1] * 10`

Reply:
Nono that's not right. Leave it as is, but maybe we need a more formal way to notify design when things don't follow their patterns